Gemfile

```diff
@@ -13,11 +13,16 @@ end
 gem 'rake', :group => [:development, :test]
 gem 'jira-ruby', :group => :development
 
+beaker_version = ENV['BEAKER_LOCATION'] || ENV['BEAKER_VERSION']
```

Comment:
We recently added `BEAKER_LOCATION`.  Why is `BEAKER_VERSION` preferred?  Is it a convention in Jenkins Job parameters or something?

There's no real reason to use `BEAKER_LOCATON` as far as I know, the `_LOCATION` simply comes from the existing convention in the puppet Gemfile.

Ideally there would be only one environment variable, not two.  Which one should it be?